tests/console/yast2_clone_system.pm

```diff
@@ -38,7 +38,25 @@ sub run {
 
     # Check and upload profile for chained tests
     upload_asset "/root/autoinst.xml";
-    if (script_run 'xmllint --noout --relaxng /usr/share/YaST2/schema/autoyast/rng/profile.rng /root/autoinst.xml') {
```

Comment:
To be honest I though that maybe running xmllint too won't harm. Do you know if they check exactly same things?

Reply:
No, I do not. I will add it there.